web/crux-ui/src/components/registries/registry-fields/v2-registry-field.tsx

```diff
@@ -25,12 +25,11 @@ const V2RegistryFields = (props: EditRegistryTypeProps<V2RegistryDetails>) => {
         message={formik.errors.url}
       />
 
-      <div className="mr-auto">
-        <DyoToggle
-          className="text-bright mt-8"
-          name="isPrivate"
-          nameChecked={t('private')}
-          nameUnchecked={t('public')}
+      <div className="flex mt-8">
+        <DyoLabel className="mr-2">{t('private')}</DyoLabel>
+
+        <DyoSwitch
+          fieldName="isPrivate"
           checked={formik.values.isPrivate}
```

Comment:
I prefer the `isPrivate` format but we know the types and @m8vago always corrects me to use `private` instead of `isPrivate`.